### PR TITLE
Make kexec compatible with Linux kexec

### PR DESCRIPTION
**Pull request by hugelgupf**
Trying it out now

### CODE REVIEW

**Comment by hugelgupf** [comment]
kexec works.

File: cmds/kexec/kexec.go [hunk 1]
```diff
@@ -5,95 +5,140 @@
...
+	f.BoolVar(&o.load, "l", false, "Load the new kernel into the current kernel.")
+	f.BoolVar(&o.load, "load", false, "Load the new kernel into the current kernel.")
+
+	f.BoolVar(&o.exec, "e", false, "Execute a currently loaded kernel.")
+	f.BoolVar(&o.exec, "exec", false, "Execute a currently loaded kernel.")
+	return o
+}
+
+// Syscall number for file kexec.
+const _SYS_KEXEC_FILE_LOAD = 320
```
Comment (rminnich) [inline cmds/kexec/kexec.go]: I just realized this should be in kexec_linux_amd64.go

**Comment by rminnich** [comment]
I will need to change the arg

**Comment by rminnich** [comment]
damn. sorry.

File: cmds/kexec/kexec.go [hunk 2]
```diff
@@ -5,95 +5,111 @@
...
 // Description:
-//     This is only intended to be used with kexec_load_files, not the older
-//     kexec.
+//		 Loads a kernel for later execution.
 //
 // Options:
-//     -dryrun:   do not do kexec system calls
-//     --cmdline: command line for kernel
-//     --i:       initramfs
+//     --cmdline=STRING:       command line for kernel
```
Comment (rminnich) [inline cmds/kexec/kexec.go]: personally, I think the single letter options are fine. Do we really need the long ones?

Comment (hugelgupf) [reply cmds/kexec/kexec.go]: we do to be compatible with kexec(8).

**Comment by hugelgupf** [comment]
Just updated this, merging it when Travis signs off.

**Pull request merged**